src/schemas/21-526EZ-allclaims/schema.js

```diff
@@ -489,7 +489,7 @@ const schema = {
             pattern: "([-a-zA-Z0-9\"\\/&()'.#]([-a-zA-Z0-9()'.# ])?)+$",
           },
           treatmentDateRange: {
-            $ref: '#/definitions/dateRangeFromRequired',
+            $ref: '#/definitions/dateRange',
```

Comment:
Can your remove this now too? https://github.com/department-of-veterans-affairs/vets-json-schema/blob/a58690f76c1a93e944b03b0dff7ee03a5f63e475/src/schemas/21-526EZ-allclaims/schema.js#L148

Reply:
yes!